assets/stylesheet/style.css

```diff
@@ -0,0 +1,360 @@
+body {
+  box-sizing: border-box;
+  margin: 0;
+}
+
+.nav-bar {
+  width: 95%;
+  margin: 0 auto;
+  display: flex;
+}
+
+header {
+  box-shadow: 0 1px 15px 1px rgba(69,65,78,.08);
+  padding: 0 16px;
+  display: flex;
+  align-items: center;
+  height: 56px;
+}
+
+i {
+  color: grey;
+  font-size: 24px;
+}
+
+.logo {
+  width: 40%;
+  vertical-align: top;
+  margin-left: 3%;
+}
+
+.user {
+  width: 10%;
+  vertical-align: top;
+}
+
+.header-left {
+  width: 18%;
+  display: inline-block;
+}
+
+
+.header-center {
+  width: 56%;
+    display: flex;
+    flex-direction: row;
+    flex: 1;
+}
+.header-right {
+  width: 23%;
+  display: inline-block;
+  text-align: right;
+}
+
+
+.header-right i, .header-right img  {
+  margin: 3px;
+  margin-left: 19px;
+}
+
+
+
+.search-field {
+  width: 89%;
+  border-bottom-left-radius: .3em;
+  border-top-left-radius: .3em;
+  border: .5px solid #80808063;
+  padding-left: 9px;
+}
+
+input:focus{
+  outline: none;
+}
+
+.search-button i {
+  font-size: 16px;
+  height: 18px;
+}
+
+.search-button {
+  width: 9%;
+  margin: 0 auto;
+  margin-left: -5px;
+  border-bottom-right-radius: .3em;
+  border-top-right-radius: .3em;
+  border: .5px solid #80808063;
+  padding: 1%;
+
+}
+.right-icons{
+  margin-left: 2%;
+}
+
+main {
+  margin: 2% 8%;
+}
+
+
+.main-video {
+  width: 60%;
+  margin-right: 2%;
+  float: left;
+
+}
+
+aside{
+  width: 37%;
+}
+
+.main-video, aside {
+  display: inline-block;
+}
+
+iframe {
+
+  top:0;
+  left:0;
+  width:100%;
+
+}
+
+.topPlay{
```

Comment:
The convention is to use [kebab-case](http://wiki.c2.com/?KebabCase) for CSS classes. I'm curious why you decided to change the naming convention for this class.

Reply:
I have made changes to the code, changing the camelCasing to kebab-case on both the CSS and the HTML